gufe/protocols/protocoldag.py

```diff
@@ -210,7 +210,9 @@ def _from_dict(cls, dct: Dict):
 
 
 def execute(protocoldag: ProtocolDAG, *, 
```

Comment:
unrelated, but can we change this to `execute_DAG()`, it always confuses me WRT `Unit.execute()`

Reply:
I'm fine with either name. Agree it's easily confused with `Unit.execute()`; was easier to keep straight when this was the `ProtocolDAG.execute()` method, but now not so much.

Reply:
renamed to execute_DAG